ovs/dal/hybrids/t_testdisk.py

```diff
@@ -36,7 +36,8 @@ class TestDisk(DataObject):
                    Relation('parent', None, 'children', mandatory=False)]
     __dynamics = [Dynamic('used_size', int, 5),
                   Dynamic('wrong_type', int, 5),
-                  Dynamic('updatable', int, 5)]
+                  Dynamic('updatable', int, 5),
```

Comment:
Updatable is actually used below, and updatable is actually more frequently used compared to updateable.